co2eq/car/index.js

```diff
@@ -0,0 +1,46 @@
+import {
+  ACTIVITY_TYPE_TRANSPORTATION,
+  TRANSPORTATION_MODE_CAR,
+} from '../../definitions';
+import cars from './cars.json';
+
+export const modelName = 'car';
+export const modelVersion = '1';
+export const explanation = {};
+
+
+// look up carbonIntensity by brand name
+export function carbonIntensityByBrand(brand) {
+  if (!cars[brand]) {
+    throw new Error(`Unknown brand name ${brand}`);
+  } else {
+    return cars[brand].carbonIntensity;
+  }
+}
+
+// look up carbon intensity for cars by size and type
+export function carbonIntensityGeneric(size, engineType, brand) {
```

Comment:
I would simply keep as:
```suggestion
export function carbonIntensity(size, engineType, brand) {
```